player/scripting.c

```diff
@@ -396,11 +396,10 @@ error: ;
 }
 
 const struct mp_scripting mp_scripting_cplugin = {
+    .name = "native",
```

Comment:
How about `C plugin`?

Reply:
I wanted to keep it one word without spaces

Reply:
`native` is kind of vague imo. I guess `Cplugin` is fine?

Reply:
Ok. I went with `native` because if this name is used in some generic msg or thread name we would have

`cplugin script` which seems weird `c` is too short so I went with `native`. We would have `native script` `lua script` `js scrip` and whatever else. I'm fine with `cplugin` though, still better than `SO plugin` or `DLL plugin` 

Changed.